app/main.py

```diff
@@ -1,2 +1,46 @@
+from typing import Any
+
+
 class Dictionary:
-    pass
+    def __init__(self, length: int = 0, capacity: int = 8) -> None:
+        self.length = length
+        self.capacity = capacity
+        self.load_factor = int(self.capacity * 2 / 3)
+        self.hash_table: list = [[]] * self.capacity
```

Comment:
it is incorrect way , you create a copy of one list, use list comp.

![image](https://user-images.githubusercontent.com/58142395/226094865-580eafdc-6b99-403a-b23b-24648bf1af95.png)